### PR TITLE
Dynamic Me Link

### PR DESCRIPTION
**Pull request by rizqirizqi**
This PR is modifying the "This is Me" button so that it gives feedback after being clicked and it shows the right information to the user.

hacktoberfest-checker

### CODE REVIEW

File: src/pages/User/components/PullRequests/MeLinkInfo.js [hunk 1]
```diff
@@ -6,17 +6,21 @@ export default class MeLinkInfo extends Component {
     username: PropTypes.string.isRequired
   };
 
-  storeUsernameAsMe = () =>
+  storeUsernameAsMe = () => {
     localStorage.setItem('myGithub', this.props.username);
+    this.forceUpdate()
+  }
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/MeLinkInfo.js]: Semi colon please

File: src/pages/User/components/PullRequests/MeLinkInfo.js [hunk 2]
```diff
@@ -6,17 +6,21 @@ export default class MeLinkInfo extends Component {
...
-    <div className="rounded mx-auto mt-16 overflow-hidden w-5/6 lg:w-1/2 mt-4">
+  render = () => {
+    let storeUsernameBtn = (
       <button
         className="bg-mid-purple text-white hover:bg-light-blue hover:text-mid-blue mx-auto mt-2 h-8 border-none pointer rounded-sm px-4 block saveUser"
         onClick={this.storeUsernameAsMe}
       >
         This is Me
       </button>
+    )
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/MeLinkInfo.js]: Semi colon please

File: src/pages/User/components/PullRequests/MeLinkInfo.js [hunk 3]
```diff
@@ -29,6 +33,30 @@ export default class MeLinkInfo extends Component {
         </a>{' '}
         on this device.
       </p>
-    </div>
-  );
+    )
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/MeLinkInfo.js]: Semi colon please

File: src/pages/User/components/PullRequests/MeLinkInfo.js [hunk 4]
```diff
@@ -29,6 +33,30 @@ export default class MeLinkInfo extends Component {
...
+    const savedUsername = localStorage.getItem('myGithub')
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/MeLinkInfo.js]: Semi colon please

File: src/pages/User/components/PullRequests/MeLinkInfo.js [hunk 5]
```diff
@@ -29,6 +33,30 @@ export default class MeLinkInfo extends Component {
...
+    if (savedUsername === this.props.username) {
+      storeUsernameBtn = null
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/MeLinkInfo.js]: Semi colon please

**Comment by jenkoian** [comment]
Minor feedback on some missing semi colons, but works great, thanks!

**Comment by rizqirizqi** [comment]
Done~ 👍

**Comment by jenkoian** [comment]
Thanks!

**Pull request merged**